sdk/typescript/packages/mix-fetch/src/index.ts

```diff
@@ -40,3 +40,17 @@ export const mixFetch: IMixFetchFn = async (url, args) => {
   // execute user request
   return instance.mixFetch(url, args);
 };
+
+/**
+ * Stops the usage of mixFetch and disconnect the client from the mixnet.
+ */
+export const disconnectMixFetch = async (): Promise<void> => {
+  // JS: I'm ignoring this lint (no-else-return) because I want to explicitly state
+  // that `__mixFetchGlobal` is definitely not null in the else branch.
+  if (!window.__mixFetchGlobal) {
+    return Promise.reject(new Error("mixFetch hasn't been setup"));
```

Comment:
You either need to wrap the whole method in `new Promise((resolve, reject) => ...)` or you can just throw an error:

```suggestion
    throw new Error("mixFetch hasn't been setup");
```

Typescript transpiles the `throw` part into a `reject(new Error(...))`.